FirstDayAtSchool.java

```diff
@@ -7,7 +7,12 @@ public static void main(String args[])
    {
       System.out.println("Hello World");
       System.out.println("Hello World !!!");
-      
+      target =- num;  // Noncompliant; target = -3. Is that really what's meant?
+		target =+ num;
+        return null;
+        /*if (i > 10) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)